docs/_config.yml

```diff
@@ -6,8 +6,8 @@ kramdown:
 
 # These allow the documentation to be updated with new releases of Alluxio.
 ALLUXIO_RELEASED_VERSION: 1.2.0
-# This would be "master" for the master branch, 1.1.1 for the 1.1.1 release, and
-# 1.1.1-RC2 for the second 1.1.1 release candidate.
+# This would be "master" for the master branch, 1.1 for the 1.1.1 release, and
+# 1.1 for the second 1.1.1 release candidate.
```

Comment:
Could this be reworded to:

`For example, "master" for the master branch, and 1.1 for 1.1.x releases and release candidates`